server/src/test/java/org/eclipse/lsp/cobol/core/model/tree/NodeTest.java

```diff
@@ -33,11 +33,11 @@
   @Test
   void getDepthFirstStream() {
     Node rootNode = new RootNode(LOCATION);
-    Node firstProg = new ProgramNode(LOCATION);
+    Node firstProg = new ProgramNode(LOCATION, null, null);
```

Comment:
Can we put another constructor, so no need to add those "NULL"s.  

```
public ProgramEndNode(Locality locality) {
this(locality, null, null);
  }
```

Reply:
Yes, I'll make it with package visibility for tests only.